src/Akka.Persistence.Redis.Tests/RedisJournalPerfSpec.cs

```diff
@@ -37,6 +38,9 @@ class = ""Akka.Persistence.Redis.Journal.RedisJournal, Akka.Persistence.Redis""
 
         public RedisJournalPerfSpec(ITestOutputHelper output, RedisFixture fixture) : base(Config(fixture, Database), nameof(RedisJournalPerfSpec), output)
         {
+            EventsCount = 1000;
+            ExpectDuration = TimeSpan.FromMinutes(10);
```

Comment:
Try to keep this low. This can tie up unit test runner on the server if something broke and causes each test waits 10 minutes before deciding that it failed, and that is a big resource waste.

Reply:
Okay